addon/templates/components/polaris-drop-zone/file-upload.hbs

```diff
@@ -15,9 +15,7 @@
     <PolarisStack @vertical={{true}} @spacing="tight">
       {{#if (eq this.type "file")}}
         <img class={{this.imageClasses}} src={{this.fileUpload}} alt="">
-      {{/if}}
-
-      {{#if (eq this.type "image")}}
+      {{else if (eq this.type "image")}}
```

Comment:
lol, not quite sure how these ended up falling under the `polaris-stack` refactor 🙈😛